draft-ietf-scitt-architecture.md

```diff
@@ -614,103 +624,103 @@ The retrieved Receipt may be embedded in the corresponding COSE_Sign1 document i
 
 # Privacy Considerations
 
-Unless advertised by the TS, every Issuer should treat its Claims as public. In particular, their Envelope and Statement should not carry any private information in plaintext.
+Unless advertised by a Transparency Service, every Issuer should treat Signed Statements it registered (rendering them Transparent Statements) as public. In particular, Signed Statement's Envelopes and Statement payload should not carry any private information in plaintext.
 
 # Security Considerations
 
-On its own, verifying a Transparent Claim does not guarantee that its Envelope or contents are trustworthy---just that they have been signed by the apparent Issuer and counter-signed by the
-TS. If the Verifier trusts the Issuer, it can infer that the Claim was issued with this Envelope and contents, which may be interpreted as the Issuer saying the Artifact is fit for its intended purpose. If the Verifier trusts the TS, it can independently infer that the Claim passed the TS Registration policy and that has been persisted in the Registry. Unless advertised in the TS Registration policy, the Verifier should not assume that the ordering of Transparent Claims in the Registry matches the ordering of their issuance.
+On its own, verifying a Transparent Statement does not guarantee that its Envelope or contents are trustworthy---just that they have been signed by the apparent Issuer and counter-signed by the
```

Comment:
Use of 'trustworthy' seems overly ambiguous. Does it mean the contents of the Transparent Statement are opaque to the TS? Does that further imply the contents could be privacy protected in some way including encrypted? If the TS provides registration assurances but contents are allowed to be opaque. Then wouldn't it make more sense to call it a "Registered Statement" instead of a "Transparent Statement"?

Reply:
Thanks @nedmsmith 
There's a combination of goal and boundary. 
The goal of SCITT is to provide verified registration, based on the identity and a registration policy. This is why you see the authors focus on the notary concept. The notary doesn't evaluate the validatiy of the document that's being notarized. Rather the notary evaluates the trustworthiness of the identities signing. It's up to a consumer to decide if the content represents the info they can act upon. However, if you can't trust the issuer, the content is no different than a piece of info found "on the street", or "on the web".

The boundary, is related to SCITT doesn't evaluate the content. 

For the Registered vs. Transparent Statement, @kaywilliams did open #20, which was duped to #8

FWIW, I do see the concern with calling everything "transparent", when the contents may be opaque. Lets see if we can continue the conversation in #8 regarding terminology.